lib/elixir/src/elixir_expand.erl

```diff
@@ -1008,6 +1023,20 @@ format_error({op_ambiguity, Name, Arg}) ->
     "\"~ts ~ts\" looks like a function call but there is a variable named \"~ts\", "
     "please use explicit parentheses or even spaces",
   io_lib:format(Message, [Name, 'Elixir.Macro':to_string(Arg), Name]);
+format_error({invalid_clauses, Name}) ->
+  SupportMessage =
+    case Name of
+      'case' ->
+        "If you are trying to use the \"case\" structure, it should be similar to:\n"
+        "    case value do"
+        "      pattern -> :ok"
+        "    end";
+
+      _ ->
+        "Clauses are only valid for case, cond, fn, receive, else, catch or rescue."
+    end,
+  io_lib:format("invalid clauses with -> operator passed to local function \"~ts\". ~s",
```

Comment:
What do you think about: "the function "~ts" cannot handle clauses with the -> operator because it is not macro. Please make sure you are invoking the proper name and that it is a macro"?